templates/crate/details.html

```diff
@@ -131,61 +128,61 @@
 
             <div class="pure-u-1 pure-u-sm-17-24 pure-u-md-19-24 package-details" id="main">
                 {# If the release is not a library #}
-                {%- if details.is_library == Some(false) -%}
+                {%- if is_library == Some(false) -%}
                     <div class="warning">
-                        {{ details.name }}-{{ details.version }} is not a library.
+                        {{ self.name }}-{{ version }} is not a library.
```

Comment:
With rinja v0.3.2 you now can `%s/self.name/name/g`, can't you?

Reply:
Yes I can!